Makefile

```diff
@@ -190,9 +192,13 @@ TEST_NAMESPACE?="e2e-test-${RANDOM_SUFFIX}"
 test-e2e-olm: DEPLOYMENT_NAMESPACE="${TEST_NAMESPACE}"
 test-e2e-olm: $(GO_JUNIT_REPORT) $(JUNITMERGE) $(JUNITREPORT) junitreportdir
 	TEST_NAMESPACE=${TEST_NAMESPACE} hack/test-e2e-olm.sh
-	$(JUNITMERGE) $$(find $$JUNIT_REPORT_OUTPUT_DIR -iname *.xml) > $(JUNIT_REPORT_OUTPUT_DIR)/junit.xml
+	echo "Complete e2e olm test"
+	$(JUNITMERGE) $$(find $$JUNIT_REPORT_OUTPUT_DIR -iname "*.xml") > $(JUNIT_REPORT_OUTPUT_DIR)/junit.xml
```

Comment:
Is this temporary or to be replaced by `merge-junit` target?

Reply:
This is temporary and will be removed in the final version. I added this because sometimes I forgot to remove the `tmp` directory after each iteration.